src/components/BrowseMenu/BrowseMenu.jsx

```diff
@@ -1,21 +1,49 @@
-import React, { Component } from 'react';
-import './BrowseMenu.css'
+import React, { useState } from 'react';
+import './BrowseMenu.css';
 
-class BrowseMenu extends Component {
-    render() {
-        return (
-            <div className='browseMenu'>
-                <div className='browseMenu__title'>Browse our menu</div>
-                <div className='browseMenu__text'>Use our menu to place an order online, or <span className="tooltip">phone<span className="tooltiptext">+375775778</span></span> our store to place a pickup order. Fast and fresh food.</div>
-                <div className='browseMenu__buttons'>
-                    <button className='browseMenu_buttons-desert' type="button">Desert</button>
-                    <button className='browseMenu_buttons-dinner' type="button">Dinner</button>
-                    <button className='browseMenu_buttons-breakfast' type="button">Breakfast</button>
-                </div>
-    
+const BrowseMenu = ({ changeCategory }) => {
+    const [selectedCategory, setSelectedCategory] = useState('Dessert');
```

Comment:
You don't need to have this state as keep selected category into the parent component